queries/currency.py

```diff
@@ -21,14 +21,14 @@
 
 """
 
+# TODO: Switch from using apis.is
```

Comment:
I agree. That project is dead, hasn't seen any love in over 4 years, and apparently is not being maintained based on the accumulation of unresolved issues and ignored pull requests.

We should be able to talk directly Arion, Landsbankinn or Seðlabankinn (probably best).

The apis.is code is here, should contain the API endpoint URLs we need:
https://github.com/apis-is/apis/tree/master/endpoints/currency